lib/screens/classes/classes_screen.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:kabod_app/core/model/classes_model.dart';
+import 'package:kabod_app/core/presentation/constants.dart';
+import 'package:kabod_app/core/repository/classes_repository.dart';
+import 'package:kabod_app/screens/commons/appbar.dart';
+import 'package:kabod_app/screens/commons/dividers.dart';
+import 'package:kabod_app/screens/commons/reusable_card.dart';
+import 'package:provider/provider.dart';
+
+class ClassesScreen extends StatefulWidget {
+  @override
+  _ClassesScreenState createState() => _ClassesScreenState();
+}
+
+class _ClassesScreenState extends State<ClassesScreen> {
+  Stream<List<Classes>> _classesStream;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  void initState() {
+    super.initState();
+    _classesStream = context.read<ClassesRepository>().getClasses();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(
```

Comment:
Don't use an appBar over here or you might open the Schedule screen over and over again and remember what happens to the RouteStack if it happens. Your Route Stack will have several layers of the same screen.

Also, maybe add a back button over here so you can go back to the menu